src/main/drivers/accgyro/accgyro.h

```diff
@@ -39,12 +39,18 @@
 #define MPU_I2C_INSTANCE I2C_DEVICE
 #endif
 
-#define GYRO_HARDWARE_LPF_NORMAL       0
-#define GYRO_HARDWARE_LPF_EXPERIMENTAL 1
-#define GYRO_HARDWARE_LPF_1KHZ_SAMPLE  2
+typedef enum {
+    GYRO_HARDWARE_LPF_NORMAL,
+    GYRO_HARDWARE_LPF_1KHZ_SAMPLE,
+#ifdef USE_GYRO_DLPF_EXPERIMENTAL
+    GYRO_HARDWARE_LPF_EXPERIMENTAL
+#endif
+} gyroHardwareLpf_e;
```

Comment:
This change requires a version increment for `gyroConfig`, to make sure the settings are reset in case one of the swapped modes was configured.

Reply:
I can increment the version but the only risk is somebody who is already using 4.0 master and happened to use the one of the unusual values.  For everybody else the eeprom version change will give them a full reset anyway.

Reply:
I think we should be consistent here, and increment the parameter group version for all changes that affect parameter groups - also for the sake of users testing with different versions. Plus this change has the potential to badly affect the flight behaviour if the modes are swapped, so erring on the side of caution is probably right.